config/samples/ccruntime/default/kustomization.yaml

```diff
@@ -8,10 +8,9 @@ resources:
 
 images:
 - name: quay.io/confidential-containers/reqs-payload
-  newTag: latest
+  newTag: 12e8fbc72e927f9d684efe2e1214937f8b0544fc
 - name: quay.io/kata-containers/kata-deploy
-  newName: quay.io/kata-containers/kata-deploy-ci
-  newTag: kata-containers-latest
+  newTag: 3.9.0
```

Comment:
I think your third commit accidentally removes the `v`.

Reply:
(although it should have been squashed to the previous commit. @niteeshkd could you please fix the `newTag` in the commit it's introduced in?)